README.md

```diff
@@ -0,0 +1,13 @@
+### (N3) Assignment 3 Pull Request
+
+##  1. Five questions using JavaScript
+- Question one, Do you have an umbrella?
+- Question two, Do you like sunny weather?
+- Question three, Did I do this assignment right?
+- Question four, I am thinking of a number.
+- Question five, Would you like to play a game?
+
+## Author of work Steven Bachtel
+# Contact information
```

Comment:
Keep in mind that this is publicly accessible info. Nothing wrong, but just letting you know.